instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/ApiGatewayProxyRequest.java

```diff
@@ -42,9 +45,16 @@ static ApiGatewayProxyRequest forStream(final InputStream source) throws IOExcep
     return new CopiedApiGatewayProxyRequest(source);
   }
 
+  private static final Function<Map.Entry<String, List<String>>, String> EXTRACTOR =
```

Comment:
Consider defining a method and using a method reference

Reply:
Good idea!